terminateTestProcesses.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/sh
+
```

Comment:
set -ex ?

Reply:
not here, don't want to exit at first failed shell command